lib/routes.js

```diff
@@ -1,9 +1,19 @@
 const express = require('express');
+const { log } = require('./log');
+
+const ApiError = require('./models/ApiError');
 
 const routes = express();
 
 routes.post('/debug/test', (req, res) => {
   return res.json({ test: 'ok' });
 });
 
+routes.post('/debug/error', () => {
```

Comment:
It seems this route might doing better only in test environment

Reply:
its just sample route will be removed after "real one" will be introduced